mise.toml

```diff
@@ -0,0 +1,3 @@
+[tools]
```

Comment:
isn't the point of these tools (mise, proto, etc) to not need tool-specific configs in the repo?

the packageManager field in the root package.json should be sufficient, yeah?
and then an engines entry?

Reply:
> isn't the point of these tools (mise, proto, etc) to not need tool-specific configs in the repo?

Whether or not this is true, it's an accident that this got pushed.